### PR TITLE
Changed issue label links in contributing.md

### PR DESCRIPTION
**Pull request by randomnoise**
"good first issue" link was broken because of white spaces, this commit simply fixes that.

### CODE REVIEW

File: doc/contributing.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ What can I do?
 You're in luck. There's various sources for tasks:
 
  - Have a look at the [issue tracker](https://github.com/sfttech/openage/issues), especially issues with labels:
-   - [good first issue](https://github.com/SFTtech/openage/labels/good first issue)!
+   - [good first issue](https://github.com/SFTtech/openage/labels/good%20first%20issue)!
```
Comment (simonsan) [inline doc/contributing.md]: Could you exchange the link with this one, as this states explicitly that is has to be an open *issue* `https://github.com/SFTtech/openage/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22`

File: doc/contributing.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ What can I do?
...
    - [just do it](https://github.com/SFTtech/openage/labels/just%20do%20it)!
```
Comment (simonsan) [inline doc/contributing.md]: While editing this document, could you also exchange this link with this one, as this states explicitly that is has to be an open issue
`https://github.com/SFTtech/openage/issues?q=is%3Aissue+is%3Aopen+label%3A%22just+do+it%22`

File: doc/contributing.md (hunk 1 above)
Comment (randomnoise) [reply doc/contributing.md]: It's done.

File: doc/contributing.md (hunk 2 above)
Comment (randomnoise) [reply doc/contributing.md]: Done. 👍

**Comment by simonsan** [comment]
Perfect, thank you!

**Pull request merged**